modules/cnv-add-disk-to-vm.adoc

```diff
@@ -0,0 +1,35 @@
+// Module included in the following assemblies:
+//
+// * cnv/cnv_users_guide/cnv-edit-vms.adoc
+// * cnv/cnv_users_guide/cnv-editing-vm-template.adoc
+
+// Establishing conditionals so content can be re-used for editing VMs
+// and VM templates.
+
+ifeval::["{context}" == "cnv-edit-vms"]
+:object: virtual machine
+:object-gui: Virtual Machines
+endif::[]
+
+ifeval::["{context}" == "cnv-editing-vm-template"]
+:object: virtual machine template
+:object-gui: Virtual Machine Templates
+endif::[]
+
+[id="cnv-vm-add-disk_{context}"]
+
+= Adding a virtual disk to a {object}
+
+.Procedure
+
+. From the *{object-gui}* tab, select your {object}.
+. Select the *Disks* tab.
+. Click *Add Disks* to open the *Add Disk* window.
+. In the *Add Disk* window, specify *Source*, *Name*, *Size*, *Interface*, and *Storage Class*.
+. Use the drop-down lists and check boxes to edit the disk configuration.
```

Comment:
I think this step could be rewritten to be clearer.

Nit: I think this should start with what you're actually doing and end with how you're doing it ("Edit the disk configuration by _____"). 

Nit 2: "Use the drop-down lists and check boxes" is vague. I think "edit the disk configuration" *might* also be too vague.

Reply:
Sent @ousleyp an email to clarify. Will follow-up on 1/7/20.

Reply:
Pan said "The first comment was just about reordering the phrase, so you could even say "Edit the disk configuration by using the drop-down lists and check boxes."

I will do this.

Reply:
Done. Also added intro sentence to procedure.